src/include/iscsi-lio-server/widgets.rb

```diff
@@ -1445,8 +1448,24 @@ def handleClient(key, event)
 
     def validateClient(key, event)
       event = deep_copy(event)
-      ret = true
-      ret
+      clients = Convert.convert(UI.QueryWidget(:clnt_table, :Items),
+                                :from => "any",
+                                :to   => "list <term>"
+                                )
+      if clients.empty?
+        return false if !Popup.AnyQuestion(
```

Comment:
Well, question in dialog say "Really want to continue without client access?". So for me more intuitive is that positive answer is "Yes" and dialog should return true when user click on continue.